sky/tools/create_macos_framework.py

```diff
@@ -152,26 +164,29 @@ def process_framework(dst, args, fat_framework, fat_framework_binary):
   # Zip FlutterMacOS.framework.
   if args.zip:
     filepath_with_entitlements = ''
-    filepath_without_entitlements = 'FlutterMacOS.framework/Versions/A/FlutterMacOS'
+
+    framework_dst = os.path.join(dst, 'FlutterMacOS.framework')
+    filepath_without_entitlements = 'Versions/A/FlutterMacOS'
 
     embed_codesign_configuration(
-        os.path.join(dst, 'entitlements.txt'), filepath_with_entitlements
+        os.path.join(framework_dst, 'entitlements.txt'),
+        filepath_with_entitlements
     )
 
     embed_codesign_configuration(
-        os.path.join(dst, 'without_entitlements.txt'),
+        os.path.join(framework_dst, 'without_entitlements.txt'),
         filepath_without_entitlements
     )
     subprocess.check_call([
         'zip',
         '-r',
         '-y',
         'FlutterMacOS.framework.zip',
-        'FlutterMacOS.framework',
+        '.',
         'entitlements.txt',
         'without_entitlements.txt',
```

Comment:
Right we would remove the metadata from the inner layer of the zip, and move them to the outer layer of the zip, so we will remove L186-187.

Reply:
duplicated the metadata to simplify removing the outer zip later.

Reply:
That will complicate the logic in several places. As the outer zipped is expected to be removed ~2weeks from now I'll hard code the name in recipes for now.